src/system/data-schedule-import.md

```diff
@@ -31,16 +31,23 @@ After each scheduled import job, a reindex operation is performed automatically.
 
    - **Entity Type** — Set to one of the following: `Products` or `Customers`
 
+      For the `Advanced Pricing`, `Products`, `Customers and Addresses (single file)`, `Stock Sources` entity’s types are displayed such import behaviors as: Add/Update, Replace, Delete.
+
+      For the `Customer Finances`, `Customers Main File`, `Customers and Addresses` entity’s types are displayed such import behaviors as: Add/Update Complex Data, Delete Entities, Custom Action.
```

Comment:
This should be a note callout and needs some tweaking on grammar and formatting.

```suggestion
      {:.bs-callout-info}
      For the _Advanced Pricing_, _Products_, _Customers and Addresses (single file)_, and _Stock Sources_ entity types, these additional import behaviors displayed: Add/Update, Replace, and Delete. For the _Customer Finances_, _Customers Main File_, and _Customers and Addresses_ entity types, these additional import behaviors displayed: Add/Update Complex Data, Delete Entities, and Custom Action.
```